corehq/form_processor/models.py

```diff
@@ -1603,6 +1603,19 @@ def location(self):
         from corehq.apps.locations.models import SQLLocation
         return SQLLocation.objects.get_or_None(supply_point_id=self.case_id)
 
+    @property
+    def sql_location(self):
+        return self.location
+
+    @property
+    @memoized
+    def sql_product(self):
+        from corehq.apps.products.models import SQLProduct
+        try:
+            return SQLProduct.objects.get(domain=self.domain, product_id=self.entry_id)
+        except SQLProduct.DoesNotExist:
+            return None
```

Comment:
Could this use `get_or_None` like the `location` property above?
```suggestion
        return SQLProduct.objects.get_or_None(domain=self.domain, product_id=self.entry_id)
```

Reply:
that's actually a custom model manager method on `SQLLocation`, not a Django models thing.